src/Compilers/CSharp/Test/Syntax/Parsing/ParserErrorMessageTests.cs

```diff
@@ -2586,16 +2585,25 @@ static void Main()
 }
 ";
 
-            ParseAndValidate(test,
-    // (10,13): error CS7014: Attributes are not valid in this context.
-    //             [ObsoleteAttribute(message)] [ObsoleteAttribute(message)] int x,
-    Diagnostic(ErrorCode.ERR_AttributesNotAllowed, "[ObsoleteAttribute(message)]"),
-    // (10,42): error CS7014: Attributes are not valid in this context.
-    //             [ObsoleteAttribute(message)] [ObsoleteAttribute(message)] int x,
-    Diagnostic(ErrorCode.ERR_AttributesNotAllowed, "[ObsoleteAttribute(message)]"),
-    // (11,13): error CS7014: Attributes are not valid in this context.
-    //             [ObsoleteAttribute(message)] int y
-    Diagnostic(ErrorCode.ERR_AttributesNotAllowed, "[ObsoleteAttribute(message)]"));
+            CreateCompilationWithMscorlib(test).VerifyDiagnostics(
+                // (10,13): error CS7014: Attributes are not valid in this context.
+                //             [ObsoleteAttribute(message)] [ObsoleteAttribute(message)] int x,
+                Diagnostic(ErrorCode.ERR_AttributesNotAllowed, "[ObsoleteAttribute(message)]").WithLocation(10, 13),
+                // (10,42): error CS7014: Attributes are not valid in this context.
+                //             [ObsoleteAttribute(message)] [ObsoleteAttribute(message)] int x,
+                Diagnostic(ErrorCode.ERR_AttributesNotAllowed, "[ObsoleteAttribute(message)]").WithLocation(10, 42),
+                // (11,13): error CS7014: Attributes are not valid in this context.
+                //             [ObsoleteAttribute(message)] int y
+                Diagnostic(ErrorCode.ERR_AttributesNotAllowed, "[ObsoleteAttribute(message)]").WithLocation(11, 13),
+                // (9,25): error CS1593: Delegate 'Action<int>' does not take 2 arguments
```

Comment:
If you'd like.  I didn't want to change any test inputs as part of this change.  I was worried that might make it so something important was missed.